misc/make_changelog.py

```diff
@@ -33,11 +33,8 @@ def format(c):
 
     for i, c in enumerate(commits):
         s = format(c)
-        if c in commits_with_tags:
-            if i == 0:
-                continue
```

Comment:
Not sure why it skips the first tagged commit, but it should be included in the changelog. For example:

```
change 1  [v0.8.10]
change 2
change 3
change 4  [v0.8.9]
```
The resulted changelog should be 
```
change 1
change 2
change 3
```

Reply:
I guess it was used to skipping version bump commits? @strongoier 
but yea it's okay to keep them in the full changelog as well.

Reply:
If that is the case we can make the parsing more explicit: only keep commits with tags, and don't include tags in commits like bump version.

Reply:
I don't really remember that because that code was there before I made changes. But it seems that our changelog doesn't miss any commit.